### ISSUE TITLE
Dictionary learning using tensorflow

### ISSUE THREAD

**Issue by chiseo36** [tensorflow/tensorflow#50368]
Hii, experts. I am looking for the implementation of dictionary learning using Tensorflow. Are there any API for the same.Please suggest.

**Comment by Saduf2019** [comment]
@chiseo36 
Please go through this example [here](https://keras.io/examples/vision/metric_learning/), as metric learning is a form of dictionary learning.
Kindly move this issue to close status and in case of any further queries open an stackoverflow/tf discussion forum issue as this is not a bug or a feat request.
